llvm/lib/Analysis/CMakeLists.txt

```diff
@@ -169,13 +169,12 @@ add_llvm_component_library(LLVMAnalysis
   BogoroditskayaLoopAnalysis.cpp
   MoiseevAnalysis.cpp
   MoiseevLoopAnalysis.cpp
-
   SelivanovskayaAnalysis.cpp
   SelivanovskayaLoopAnalysis.cpp
   DVorobyovAnalysis.cpp
   DVorobyovLoopAnalysis.cpp
-  SazanovAnalysis.cpp
-  SazanovLoopAnalysis.cpp
+  ErmolaevAnalysis.cpp
```

Comment:
@J1nsei, why do you replace some other pass? I'd expect you just to add your passes

Reply:
Github desktop partially updated local rep and my build broke. I had to roll back the changes to get everything working. I couldn't merge the changes in the program, so I was hoping to do it on the site.